test/41-device-api-v1.spec.ts

```diff
@@ -841,15 +841,15 @@ describe('SupervisorAPI [V1 Endpoints]', () => {
 				configSetStub = stub(config, 'set').callsFake(configSetFakeFn);
 			});
 
-			after(async () => {
+			after(() => {
```

Comment:
You can also probably squash these changes into 18f03a0

Reply:
Squashed into 12ce1fd2c23f606040afcc27f2b1e4c5dfd48590